src/server/api/data.ts

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+import "../config";
+
+interface Coordinates {
+  latitude: number;
+  longitude: number;
+}
+
+interface PostcodeResponse {
+  result: { latitude: number; longitude: number };
+}
+
+interface TicketmasterResponse {
+  _embedded: any;
```

Comment:
Why not type `_embedded` here as well?  At minima, this should indicate that there is a `events` prop that is an array of _something_. It is ok for the _something_ to be `unknown` for now. When we start transforming the TM data into a custom types for our React frontend, we can type this better.
<img width="528" alt="Screenshot 2021-09-13 at 21 59 13" src="https://user-images.githubusercontent.com/9220319/133155635-c4ba3032-881a-41a8-85a0-90da92e339c2.png">